substrate/primitives/core/src/const_hex2array.rs

```diff
@@ -18,7 +18,19 @@
 //! Provides a const function for converting a hex string to a `u8` array at compile time, when used
 //! in the proper context.
 
-/// Util that generates array from (static) string literal.
+/// Provides a const array from given string literal.
+///
+/// Valid characters are `[0-9a-fA-F]`, and the hex string should not start
+/// with the `0x` prefix.
+#[macro_export]
+macro_rules! hex2arr {
+	($input:expr) => {{
+		const PUBLIC_BYTES: [u8; $input.len() >> 1] = $crate::const_hex2array::hex2array($input);
+		PUBLIC_BYTES
```

Comment:
```suggestion
		const BYTES: [u8; $input.len() >> 1] = $crate::const_hex2array::hex2array($input);
		BYTES
```
Nit: I'd rename it to just `BYTES` as this is a generic macro which can be used for not just public keys

Reply:
done in: 0f7cccc